scripts/reindex.py

```diff
@@ -211,11 +213,11 @@ def conda_data_wrapper(args):
         # write summary map of pypi package name to conda channel/name
         (PUBLIC / "conda.json").write_text(json.dumps(CONDA_INDEX, indent=2))
 
-        # update the main index (summary) with the conda versions basic sorting
-        # should match the method used in scripts/bigquery.py for PyPI
-        # versions. If you need fancy sorting downstream use 'pacakging'.
+        # update the main index (summary) with the conda versions
+        # de-dupe and sort as in scripts/bigquery.py
         for pkg in PYPI_INDEX:
-            pkg["conda_versions"] = sorted(data.get(pkg["name"], {}).get("versions", []))
+            versions = data.get(pkg["name"], {}).get("versions", [])
+            pkg["conda_versions"] = sorted(set(versions), key=lambda v: Version(v))
```

Comment:
How about, so it is sorted in descending fashion like we will use them :) 
```suggestion
            pkg["conda_versions"] = sorted(set(versions), key=version.Version, reverse=True)
```

Reply:
Thanks - much cleaner without the lambda 🤦